Rlog/View/WorkSpace/WorkSpaceCreateCreatingScheduleView.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  WorkSpaceCreateCreatingScheduleView.swift
+//  Rlog
+//
+//  Created by 송시원 on 2022/10/24.
+//
+
+import SwiftUI
+
+struct WorkSpaceCreateCreatingScheduleView: View {
+    @ObservedObject var viewModel:  WorkSpaceCreateCreatingScheduleViewModel
+    @Binding var isShowingModal: Bool
+    @Binding var scheduleList: [Schedule]
+    
+    
+    
+    var body: some View {
```

Comment:
여기 빈 공간에 혹시 들어갈 코드가 있을가요?
없으시다면 1칸만 띄우는 것을 추천드립니다!

```suggestion
    @Binding var scheduleList: [Schedule]
    
    var body: some View {
```